Samples/luis/README.md

```diff
@@ -0,0 +1,105 @@
+
+![Get the programmatic key](images/programmatic-key.png)
+
+Set the `LUIS_SUBSCRIPTION_KEY` environment variable to this programmtic key to continue.
+
+## Cognitive Services: LUIS Authoring Sample
+
+We provide two samples:
+
+- The booking sample who is a complete example on how to create an app, create intents, utterances, training and publishing the app
+- The management example shows management operations: import or export an app/version, listing the different versions of an app, cloning a version, etc.
+
+## Cognitive Services: LUIS Runtime Sample
+
+#### LUIS Application
+
+The first step to using LUIS is to create or import an application. Go to the home page, [www.luis.ai](https://www.luis.ai/), and log in. After creating your LUIS account you'll be able to Import an Existing Application where can you can select a local copy of the LuisApp.json file an import it.
+
+![Import an Existing Application](images/prereqs-import.png)
+
+If you want to test this sample, you have to import the pre-build [LuisApp.json](LuisApp.json) file to your LUIS account.
+
+Once you imported the application you'll need to "train" the model ([Training](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/train-test)) before you can "Publish" the model in an HTTP endpoint. For more information, take a look at [Publishing a Model](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/publishapp).
+
+Finally, edit the [luisRuntime.js](luisRuntime.js) file and update the attribute placeholders with the values corresponding to your Application and Azure Region where the application was deployed.
+
+#### Where to find the Application ID and Subscription Key
+
+You'll need these two values to configure the LuisDialog through the LuisModel attribute:
+
+1. Application ID
+
+    You can find the App ID in the LUIS application's settings.
+
+    ![App Settings](images/prereqs-appid.png)
+
+2. Subscription Key and Azure Region
+
+    Click on the Publish App link from the top of the LUIS application dashboard. Once your app is published, copy the Region and Key String from *Starter_Key* from the Endpoints table on the Publish App page.
+
+    ![Programmatic API Key](images/prereqs-apikey.png)
+
+
+### Code Highlights
+
+One of the key problems in human-computer interactions is the ability of the computer to understand what a person wants, and to find the pieces of information that are relevant to their intent. In the LUIS application, you will bundle together the intents and entities that are important to your task. Read more about [Planning an Application](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/plan-your-app) in the [LUIS Docs](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/).
+
+Once your model is set, you can invoke the LUIS Runtime API to analize user input and obtain its intent and possible entities.
```

Comment:
"analize" -> "analyze"

Reply:
Do I have permissions to fix the same issue in Python's samples?